src/actions-on-google.ts

```diff
@@ -515,12 +541,20 @@ export class ActionsOnGoogle {
                 ssml: [],
                 suggestions: [],
             }
+            const audioBuffers: Buffer[] = []
 
             conversation.on('data', (data: AssistantSdkResponse) => {
+                if (this.include && this.include.audioOut &&
+                    data.audio_out && data.audio_out.audio_data) {
+                    audioBuffers.push(data.audio_out.audio_data)
+                }
+
                 if (data.dialog_state_out) {
                     this._conversationState = data.dialog_state_out.conversation_state
                     if (data.dialog_state_out.supplemental_display_text &&
-                        !assistResponse.displayText.length) {
+                        (!assistResponse.displayText.length &&
```

Comment:
If we're AND-ing all these operations together, do these three need to be in nested parenthesis?

Reply:
no